scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -1629,10 +1643,10 @@ function Uninstall-Services([ref] $RestartNeeded, [bool] $LegacyInstaller) {
         Stop-Service -NoWait -ErrorAction SilentlyContinue -ErrorVariable cmdErr $MobyServiceName
         if ($?) {
             Start-Sleep -Seconds 7
-            Write-Verbose 'Stopped the IoT Edge Moby Engine service'
+            Write-Verbose "'Stopped the IoT Edge Moby Engine service $MobyServiceName"
         }
         else {
-            Write-Verbose "$cmdErr"
+            Write-Verbose "stopping IoT Edge Moby Engine service $MobyServiceName failed.  exitcode: $LASTEXITCODE error: $cmdErr"
```

Comment:
Same here, `$LASTEXITCODE` is meaningless for `Stop-Service`

Reply:
fixed per above